themes/index.ts

```diff
@@ -366,7 +424,8 @@ export const themes = {
     icon_color: "ebbcba",
     text_color: "e0def4",
     bg_color: "191724",
+    border_color: "",
   },
 };
 
-export default themes;
```

Comment:
I think we should create a Theme interface and a Themes interface with [an index signature](https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures).